lms/templates/courseware/courseware.html

```diff
@@ -206,6 +206,16 @@
                     current_score=int(entrance_exam_current_score * 100)
                 )}
             </p>
+            <script type="text/javascript">
+            $(document).ajaxSuccess(function(event, xhr, settings) {
+                if (settings.url.indexOf("xmodule_handler/problem_check") > -1) {
+                    var data = JSON.parse(xhr.responseText);
+                    if (data.entrance_exam_passed){
+                        location.reload();
```

Comment:
@ziafazal as per my understanding this PR is reloading the page after successful exam completion but not updating the score on each student attempt OR the score is already updating ?